src/core/api/index.js

```diff
@@ -124,9 +126,10 @@ export function callApi({
     ...parsedUrl.query,
     ...params,
     lang: apiState.lang,
-    // Always return URLs wrapped by the outgoing proxy.
+    // Always return URLs wrapped by the outgoing proxy, except when requested
+    // not to do so.
     // Example: http://outgoing.prod.mozaws.net/
-    wrap_outgoing_links: true,
+    wrap_outgoing_links: wrapOutgoingLinks || null,
```

Comment:
It's a bit sad that we can't pass `false` to `wrap_outgoing_links` :/

Reply:
Agreed!